src/unstuck-bridge-tx.md

```diff
@@ -0,0 +1,305 @@
+:::
+
+## Overview of the Steps in this Guide
+
+- [Step 1](#Step-1) Check if your bridge transaction is actually stuck or is just pending.
+
+- [Step 2](#step-2) Find the transaction hash for your stuck transaction.
+
+- [Step 3](#step-3) Get the data needed to unstuck the transaction (getting your proof).
+
+- [Step 4](#step-4) Use your proof to manually call your Ethereum transaction to receive your tokens.
+
+**NOTE: [Step 4](#step-4) is slightly different for each token so make sure to
+follow the correct subsection.**
+
+### Step 1
+
+Check if your transaction is actually stuck or just pending.
+
+- Connect to Polkadot.js apps: [https://polkadot.js.org/apps/rpc=wss%3A%2F%2Fws.sora2.soramitsu.co.jp#/rpc](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.sora2.soramitsu.co.jp#/rpc)
+
+- Click "Developer", then Select "RPC calls"
+  ![](/.gitbook/assets/unstucktx-rpc-call.png)
+
+- Click “author”, then Select “Ethbridge” (by default “getAccountRequests(accountId, statusFilter)” should be selected, if not, select it.
+  ![](/.gitbook/assets/unstucktx-ethbridge.png)
+
+- Ensure your account is selected underneath, click on "include option" and choose "Pending", then Submit RPC call
+  ![](/.gitbook/assets/unstucktx-include-option.png)
+
+If your result contains any transaction hashes, these ARE still pending and may take some time to go through. **If your result looks as below and produces no hashes then your SORA bridge transaction has been approved and you should continue through the guide to unstuck your transaction.**
+![](/.gitbook/assets/unstucktx-approved-continue.png)
+
+### Step 2
+
+Find the transaction hash of your approved (but stuck) transaction.
+
+- If continuing from step 1 your RPC call only needs slight adjusting, click on “include option” but this time select “Approvals Ready”, Then Submit RPC Call
+  ![](/.gitbook/assets/unstucktx-approvals-ready.png)
+
+The result is a list of all SORA bridge transactions that have been
+approved on the SORA side. Your latest bridge transaction is the last
+one (scroll down if necessary), if you have made more SORA>ETH bridge
+transactions after the stuck one, you will need to go from the bottom
+to the top of the list until you find your stuck transaction.
+
+- Copy your “stuck” transaction hash, it will look like this:
+
+`0x57533dc46a5d257b6dj0nhagb0f8s4dfedfl6cpppcoaidu99dd0sdc4def4g6b`
+
+![](/.gitbook/assets/unstucktx-stuck-tx.png)
+
+### Step 3
+
+Use your transaction hash to get the required data (proof) for step 4.
+
+- Continuing from step 2, click on “get**Account**Requests(accountId, statusFilter)” and change it to “get**Approved**Requests(requestHashes, networkId)
+
+![](/.gitbook/assets/unstucktx-get-approved-requests.png)
+
+- Click on “Add item”
+- Paste in your Approved transaction hash from step 2 instead of 0x000…
+- Click “Include Option” and leave the network id as 0
+- Submit the RPC Call
+
+![](/.gitbook/assets/unstucktx-add-approved-request.png)
```

Comment:
```suggestion
1. Continuing from step 2, click on “get**Account**Requests(accountId, statusFilter)” and change it to “get**Approved**Requests(requestHashes, networkId)

![](/.gitbook/assets/unstucktx-get-approved-requests.png)

2. Click on “Add item”.
3. Paste in your Approved transaction hash from step 2 instead of 0x000…
4. Click “Include Option” and leave the network id as 0.
5. Submit the RPC Call:

![](/.gitbook/assets/unstucktx-add-approved-request.png)

```